com.github.bcedu.valasimplehttpserver.json

```diff
@@ -0,0 +1,40 @@
+{
+    "app-id": "com.github.bcedu.valasimplehttpserver",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "juno",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.bcedu.valasimplehttpserver",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home",
```

Comment:
Does it allow you to select a folder and share it? Otherwise using Gtk.NativeFileChooser would have been enough.

Reply:
It must have access to files and to share them. I don't know very well the function of this options, I've put them because i've seen it in similar apps